test_scripts/API/SendLocation/001_SendLocation_success.lua

```diff
@@ -0,0 +1,103 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/3
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/SetInteriorVehicleData.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [SDL_RC] Set available control module settings SetInteriorVehicleData
+--
+-- Description:
+-- In case:
+-- 1) Application is registered with REMOTE_CONTROL appHMIType
+-- 2) and sends valid SetInteriorVehicleData RPC with valid parameters
+-- SDL must:
+-- 1) Transfer this request to HMI
+-- 2) Respond with <result_code> received from HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common_send_location = require('test_scripts/API/SendLocation/common_send_location')
+
+--[[ Local Variables ]]
+local request_params = {
+    longitudeDegrees = 1.1,
+    latitudeDegrees = 1.1,
+    address = {
+        countryName = "countryName",
+        countryCode = "countryName",
+        postalCode = "postalCode",
+        administrativeArea = "administrativeArea",
+        subAdministrativeArea = "subAdministrativeArea",
+        locality = "locality",
+        subLocality = "subLocality",
+        thoroughfare = "thoroughfare",
+        subThoroughfare = "subThoroughfare"
+    },
+    timeStamp = {
+        second = 40,
+        minute = 30,
+        hour = 14,
+        day = 25,
+        month = 5,
+        year = 2017,
+        tz_hour = 5,
+        tz_minute = 30
+    },
+    locationName = "location Name",
+    locationDescription = "location Description",
+    addressLines = 
+    { 
+        "line1",
+        "line2",
+    }, 
+    phoneNumber = "phone Number",
+    deliveryMode = "PROMPT",
+    locationImage = 
+    { 
+        value = "icon.png",
+        imageType = "DYNAMIC",
+    }
+}
+
+--[[ Local Functions ]]
+local function send_location(params, self)
+    local cid = self.mobileSession1:SendRPC("SendLocation", params)
+
+    local deviceID = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+    params.locationImage.value = common_send_location.getPathToSDL() .. "storage/"
```

Comment:
@Itileda What is porpose of changing params after request was sent in this case?
Maybe it make sense to move this code before request sending?

Reply:
@aderiabin , initially params contains valid mobile request parameters and I use it to send mobile request. When SDL receive it,  the same parameters will be resend to HMI except image value parameter. SDL provides to HMI full path to the file. and this one is changed and used in next row in expectation.